examples/node/spec/integration/TestRequestClient.ts

```diff
@@ -0,0 +1,12 @@
+import RequestClient from '../../lib/base/RequestClient';
+
+export default class TestRequestClient extends RequestClient {
+    constructor() {
+        super();
+    }
+
+    request(opts: object) {
+        opts.uri = opts.uri.replace("https", "http");
```

Comment:
I say we just update the `rest/*Base.ts` files to be `http` instead and then this isn't needed.

Reply:
Can delete this now and just use `RequestClient`.